examples/basic-nextjs/next.config.ts

```diff
@@ -0,0 +1,66 @@
+import type { NextConfig } from 'next';
+import createNextIntlPlugin from 'next-intl/plugin';
+
+const nextConfig: NextConfig = {
+  // Allow specifying a distinct distDir when concurrently running app in a container
+  distDir: process.env.NEXTJS_DIST_DIR || '.next',
+  
+  // Enable React Strict Mode
+  reactStrictMode: true,
+
+  // Disable the X-Powered-By header. Follows security best practices.
+  poweredByHeader: false,
+
+  // Webpack configuration to handle Node.js modules in browser environment
+  webpack: (config, { isServer }) => {
+    if (!isServer) {
+      // Fallbacks for Node.js modules when running in browser
+      config.resolve.fallback = {
+        ...config.resolve.fallback,
+        fs: false,
+        module: false,
+        path: false,
+        os: false,
+        crypto: false,
+      };
+    }
+    return config;
+  },
```

Comment:
We don't have these settings in content-sdk starter and in my poc as well, is there some reason add them?